tests/ui/lint/improper-ctypes/unsafe-binder-basic.rs

```diff
@@ -0,0 +1,11 @@
+// build-fail
+#![feature(unsafe_binders)]
+//~^ WARN the feature `unsafe_binders` is incomplete
+#![deny(improper_ctypes)]
```

Comment:
`// build-fail  comment is not necessary. And if you use `#![expect(incomplete_features)]`, the WARN will not occur.